.github/workflows/test_and_deploy.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, '3.10']
```

Comment:
I guess that version number should not be quoted? But I might be wrong.

Reply:
Well I'm not sure my self but that's how it is in the current template for npe2 plugins.

Reply:
It does need to be quoted to be interpreted correctly by GitHub Actions.